test/functionalTest/cases/0999_update_single_attr_value_by_eid/update_single_attr_by_eid.test

```diff
@@ -163,10 +163,15 @@ echo
 01. PUT /v2/entities/E1/attrs/A1/value and see it fail
 ======================================================
 HTTP/1.1 404 Not Found
-Content-Length: 0
+Content-Length: 61
+Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
+{
```

Comment:
This change in .test (and some other similar) corresponds to the following entry in CNR: "Fix: missing error payload for NotFound error in 'PUT /v2/entities/<eId>/attrs/<attrName>' and **'PUT /v2/entities/<eId>/attrs/<attrName>/value'** operations".

NTC (just informative)